### ISSUE TITLE
Disable users generates a problem to some resources

### ISSUE THREAD

**Issue by mconf-daileon** [mconf/mconf-web#483]
---

Author Name: **Fernando Bottin** (Fernando Bottin)
Original Redmine Issue: 947, http://dev.mconf.org/redmine/issues/947

---

After disable a user, the system sets the user model to nil. Because of this some calls methods present a strange behavior, soon the calls methods can't be found in a nil model.

This problem can be visualized after disable some user and enter in a space wall with any post of this user.

In the old version of mconf it's not happens, so I think look into the old version can be a good start point to correct this problem.

**Comment by mconf-daileon** [comment]
---

Original Redmine Comment
Author Name: **Leonardo Daronco** (Leonardo Daronco)
Original Date: 2014-08-04T01:09:08Z

---

Should review the app in general for possible problems with disabled users and add tests for whatever is found.

**Comment by mconf-daileon** [comment]
---

Original Redmine Comment
Author Name: **Leonardo Daronco** (Leonardo Daronco)
Original Date: 2015-03-09T14:24:56Z

---

After #1173, where we improved the permissions over disabled resources, we tested the application with different types of disabled resources and the errors found were fixed.
We now also have disabled resources being generated by "db:populate", so developers are constantly using a web portal that has disabled resources.

No more errors with disabled users seem to exist, so I'm closing this as resolved.

If more issues are found in the future we can reopen this or create new tickets.
